docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+---
+version: "3.4"
+services:
+  mongodb:
+    image: mvertes/alpine-mongo:latest
+    ports:
+      - "27017:27017"
+  web:
+    build: .
+    depends_on:
+      - mongodb
+    ports:
+      - "3000:3000"
```

Comment:
You may want to add the `command` override from below with the `wait-for-it.sh` call, so that this container will wait for mongo to be started before executing.